adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -212,16 +213,25 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
      * Perform authority validation.
      * True if the passed in authority is valid, false otherwise.
      */
-    private void validateAuthority(final URL authorityUrl, final UUID correlationId) throws AuthenticationException {
+    private void validateAuthority(final URL authorityUrl, @Nullable final String domain, final UUID correlationId) throws AuthenticationException {
```

Comment:
I think this is entirely a judgment call.

That said, I suspect the future maintainer would prefer to have information about the nullability of parameters pointed out to them explicitly, rather than go digging through the code to discover it isn't a bug